src/olympia/abuse/tests/test_admin.py

```diff
@@ -522,7 +522,7 @@ def test_detail_addon_report(self):
         assert 'Neo' in doc('.addon-info-and-previews h2').text()
         assert doc('.addon-info-and-previews .meta-abuse td').text() == '2'
         assert doc('.addon-info-and-previews .meta-rating td').text() == (
-            'Rated 2 out of 5 stars 1 review'
+            'Rated 2 out of 5 stars 1 review(s)'
```

Comment:
So removing the translation also removed the logic that would make this "review" instead of "review(s)"?

Reply:
Yes, because we no longer have a `ngettext`